debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
Since the version has not been added to the control file, I get this impression that it could be picking it up from here?

Is there any better way to manage versioning and are there any rules about the numbering? I'm thinking of generating automatic builds and would need to manually edit whichever file defines the version in order to append an incremental number (so if mu is at version 0.2 right now, I'd like to generate builds with something like `0.2-<incremental number>a` or anything else to denote it's not "production" code.)